### PR TITLE
fix(components) : fix space between header & table on firefox

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

There is a space of 6 px between header & table content

**What is the new behavior?**

There is no space

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by talend-glorieux** [comment]
Where is the change? Is it the html element change that fixes the issue? 
https://github.com/Talend/ui/pull/234/files#diff-a24714ed263cd5d3446882f7bb3fa315R154

**Comment by talend-sromain** [comment]
it's not the html element ny himself, actually, it's the display of this html element.
On FF with a span, i got the element at 6px height, when i switch it to display:block, the 'real' height given by css is applied.
So instead of setting a display block on a span, i think it's better to set a div displayed block by default

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://234.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://234.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://234.talend.surge.sh)

File: output/components.eslint.txt [hunk 1]
```diff
@@ -16,6 +16,10 @@ Scoping to packages that match 'react-talend-components'
   2:1  error  '@kadira/react-storybook-addon-info' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   7:2  error  Unexpected require()                                                                                      global-require
 
+/home/travis/build/Talend/ui/packages/components/src/List/DisplayTable/DisplayTable.component.js
```
Comment (jlame) [inline output/components.eslint.txt]: some lint to fix

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://234.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://234.talend.surge.sh)

**Pull request merged**